buildsystem/spack/deception/spack.yaml

```diff
@@ -24,9 +24,9 @@ spack:
     magma:
       require: '@2.6.2'
     coinhsl:
-      require: '@2019.05.21'
+      require: '@2023.11.17'
     ipopt:
-      require: '@3.12.10~metis+coinhsl~mumps'
+      require: '~metis+coinhsl~mumps'
```

Comment:
I believe you need to build coinhsl with Metis support. Otherwise the linear solver and by extension Ipopt will have diminished performance.

CC @nychiang @cameronrutherford